privaterelay/views.py

```diff
@@ -157,6 +158,38 @@ def waitlist(request):
     return JsonResponse({'email': email}, status=status)
 
 
+# TODO: refactor to consolidate with emails.views
+@csrf_exempt
+def metrics_event(request):
+    request_data = json.loads(request.body)
+    # TODO: set and get ga_uuid in/from add-on, instead of looking it up
+    if (
+        not request.user.is_authenticated and
```

Comment:
Does this means users need to be signed in to the website in order for events to fire?

Reply:
To send them along to GA, yes. Because we need to send them with a `ga_uuid`.

So in the add-on, I suggest we don't send any pings unless/until the user signs in.

Reply:
We fixed this by generating the GA uuid in the add-on and sending it with the metrics ping.